kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -354,12 +335,6 @@ define([
                     });
                     break;
 
-                case 'job_init_err':
-                case 'job_init_lookup_err':
-                    this.displayJobError(msgData);
-                    console.error('Error from job comm:', msg);
-                    break;
```

Comment:
Could not find any mention of these errors anywhere in the BE -- we now use `job_error` to report any job-related problems and any startup errors are handled by other code in the FE.